modules/sql/managed-instance/.bicep/nested_storageRoleAssignment.bicep

```diff
@@ -0,0 +1,17 @@
+param storageAccountName string
```

Comment:
I think your original location in the vuln. ass. folder was better.

Reply:
Reverted back to original location.